### PR TITLE
chore : match QA with PROD

### PR DESCRIPTION
**Pull request by atharvar28**

### CODE REVIEW

File: qa-covid19.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -6,26 +6,28 @@
   "versions": {
     "arborist": "quay.io/cdis/arborist:2020.03",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
+    "covid19-etl": "quay.io/cdis/covid19-etl:1.0.0",
```
Comment (paulineribeyre) [inline qa-covid19.planx-pla.net/manifest.json:9]: that's not the same version as prod, but it's ok. we are releasing a new version tomorrow anyway

File: qa-covid19.planx-pla.net/manifest.json [hunk 2]
```diff
@@ -6,26 +6,28 @@
...
+    "portal": "quay.io/cdis/data-portal:wip_covid19dashboard",
     "tube": "quay.io/cdis/tube:2020.03",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
     "spark": "quay.io/cdis/gen3-spark:2020.03",
     "hatchery": "quay.io/cdis/hatchery:2020.03",
     "wts": "quay.io/cdis/workspace-token-service:2020.03",
     "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2020.03",
-    "ambassador": "quay.io/datawire/ambassador:0.60.3"
+    "ambassador": "quay.io/datawire/ambassador:0.60.3",
+    "guppy": "quay.io/cdis/guppy:2020.03"
```
Comment (paulineribeyre) [inline qa-covid19.planx-pla.net/manifest.json:24]:
```suggestion
    "guppy": "quay.io/cdis/guppy:0.4.2"
```
to match what we're trying to get into prod: https://github.com/uc-cdis/cdis-manifest/pull/1317/files

File: qa-covid19.planx-pla.net/portal/gitops.json [hunk 3]
```diff
@@ -241,19 +199,14 @@
       }
     ],
     "guppyConfig": {
-      "dataType": "subject",
-      "nodeCountTitle": "Subjects",
-      "fieldMapping": [
-        { "field": "study_submitter_id", "name": "Study" }
-      ],
+      "dataType": "location",
```
Comment (paulineribeyre) [inline qa-covid19.planx-pla.net/portal/gitops.json:202]: i know this mistake is still in prod, but `location` isn't in the dictionary anymore, we might want to fix this file
maybe work with Di tomorrow?

Comment (atharvar28) [reply qa-covid19.planx-pla.net/portal/gitops.json:202]: 👍

File: qa-covid19.planx-pla.net/manifest.json [hunk 4]
```diff
@@ -6,27 +6,29 @@
...
-    "portal": "quay.io/cdis/data-portal:2.24.8",
+    "portal": "quay.io/cdis/data-portal:wip_covid19dashboard",
     "tube": "quay.io/cdis/tube:2020.03",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
     "spark": "quay.io/cdis/gen3-spark:2020.03",
     "hatchery": "quay.io/cdis/hatchery:2020.03",
     "wts": "quay.io/cdis/workspace-token-service:2020.03",
     "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2020.03",
     "ambassador": "quay.io/datawire/ambassador:0.60.3",
+    "guppy": "quay.io/cdis/guppy:0.4.2"
```
Comment (diw2) [inline qa-covid19.planx-pla.net/manifest.json:24]: there should be , at the end

File: qa-covid19.planx-pla.net/manifest.json [hunk 5]
```diff
@@ -136,5 +138,20 @@
     "job_images": {
       "indexing": "quay.io/cdis/indexs3client:2020.03"
     }
+  },
+  "guppy": {
+    "indices": [
+      {
+        "index": "covid19",
```
Comment (diw2) [inline qa-covid19.planx-pla.net/manifest.json:145]: Could you please add qa- prefix too all index ? 🙏

**Pull request merged**